django/thunderstore/account/api/experimental/views.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.authtoken.models import Token
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+
+class TokenExperimentalApiView(APIView):
```

Comment:
View itself could also be called something more elaborate, e.g. `CreateSessionTokenExperimentalApiView`. Quite long but more clear